<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(51, 209, 122);
border-radius:10px</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="emprunt_2">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(246, 245, 244);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>membres</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(246, 245, 244);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>livres</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="emprunt">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(246, 245, 244);
border-radius:5px
</string>
      </property>
      <property name="text">
       <string>emprunt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(246, 245, 244);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>categorie</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(246, 245, 244);
border-radius:10px</string>
   </property>
   <property name="text">
    <string>BIBLIO-TECH</string>
   </property>
   <property name="margin">
    <number>320</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>128</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #e74c3c;        /* Rouge vif */
    color: white;
    border: none;
    border-radius: 16px;
    padding: 8px 16px;
    font-weight: bold;
    min-width: 100px;
}

QPushButton:hover {
    background-color: #c0392b;        /* Rouge plus foncé au survol */
}

QPushButton:pressed {
    background-color: #a93226;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #e74c3c;
}

/* Option bonus : petite icône poubelle à gauche (si tu mets une icône) */
QPushButton {
    icon-size: 18px;
    padding-left: 12px;
}</string>
   </property>
   <property name="text">
    <string>suppimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

    background-color: #28a745;     /* Vert Bootstrap success */
    color: white;
    border: 2px solid #28a745;
    border-radius: 16px;
    padding: 10px 24px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton#btn_ajouter:hover,
QPushButton[objectName=&quot;btn_ajouter&quot;]:hover {
    background-color: #218838;
    border-color: #1e7e34;
}

QPushButton#btn_ajouter:pressed,
QPushButton[objectName=&quot;btn_ajouter&quot;]:pressed {
    background-color: #1e7e34;
</string>
   </property>
   <property name="text">
    <string>ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #007bff;     /* Bleu classique */
    color: white;
    border: 2px solid #007bff;
    border-radius: 16px;
    padding: 10px 24px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton#btn_modifier:hover,
QPushButton[objectName=&quot;btn_modifier&quot;]:hover {
    background-color: #0069d9;
    border-color: #0062cc;
}

QPushButton#btn_modifier:pressed,
QPushButton[objectName=&quot;btn_modifier&quot;]:pressed {
    background-color: #0062cc;
</string>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>611</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-radius: 16px;
background-color:grey;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>emprunt_2</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>emprunt</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
